src/pages/administration/datasources/webuntis/new.vue

```diff
@@ -51,6 +51,7 @@
 						label="Passwort"
 						:placeholder="'WebUntis Passwort'"
 						class="mt--md"
+						autocomplete="new-password"
```

Comment:
I think 
```suggestion
						autocomplete="off"
```
is semanticly better suited here, because new-password seems to indicate that submitting this form updates the currently saved login credentials.

Reply:
autocomplete off does not work for all browsers unfortunately... it's already set for the form
autocomplete="new-password" seems the only option to make it work for all browsers. 
I did some research...

Reply:
agree. Just didn't knew `new-password` before and haven't found that little piece of information.